src/release/breaking-changes/dispose.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Added missed `dispose()` for some disposable objects in Flutter
+description: >
+  'dispose()' may fail because of double disposal.
+---
+
+## Summary
+
+Missed 'dispose()' are added for some disposable objects. If some other code
+also invokes 'dispose()' for the objects, and the object is protected from double
+disposal, the second 'dispose()' will fail with error message:
+
+`Once you have called dispose() on a <class name>, it can no longer be used.` 
+
+## Background
+
+There is a rule that owner of an object should dispose it.
+
+This rule was broken in some places in Flutter Framework: owners were not disposing the disposable objects.
+The issue was fixed by adding `dispose()`. The added `dispose()` can cause failures in debug mode,
+if there is another `dispose()`, and the object is protected from double disposal.
+
+## Migration guide
+
+If you got error like `Once you have called dispose() on a <class name>, it can no longer be used.`,
+and the error is raised for your code, 
+update the code to call `dispose()' only in cases when your code created the object.
+
+Code before migration:
+
+```dart
+x.dispose();
+```
+
+Code after migration:
+
+```dart
+if (xIsCreatedByMe) {
+  x.dispose();
+}
+```
+
+If the error is for Flutter code, this means wrong `dispose()` happened first, and the error was raised
+for correct `dispose()` that happened second.
+
+You can locate the wrong `dispose()` by temporary adding `print(StackTrace.current)`
+to the body of the failed method `dispose`.
```

Comment:
It seems like you don't really need this first sentence?
If so, delete. That second sentence is super useful.

```suggestion
If the error occurs in Flutter code, `dispose()` was
called incorrectly the first time.

You can locate an incorrect call to `dispose()` by
temporary calling `print(StackTrace.current)`
in the body of the failed `dispose` method.
```

Reply:
print(StackTrace.current) makes sense if the error is for Flutter code, otherwise it is not needed.
I updated text to made it clearer.